autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterCommonService.java

```diff
@@ -103,25 +109,38 @@ public Cluster setState(Long clusterId, StateJson stateJson) {
 
     public Cluster setAutoscaleState(Long clusterId, AutoscaleClusterState autoscaleState) {
         Cluster cluster = setAutoscaleState(clusterId, autoscaleState.isEnableAutoscaling());
-        return setStopStartScalingState(cluster.getId(), autoscaleState.getUseStopStartMechanism());
+        return setStopStartScalingState(cluster.getId(), autoscaleState.getUseStopStartMechanism(),
+                cluster.getTimeAlerts() != null && !cluster.getTimeAlerts().isEmpty());
```

Comment:
Small nit: You can reduce verbosity by using `ObjectUtils.isEmpty(cluster.getTimeAleets())` which checks for `null` or `Collection#isEmpty` automatically

Reply:
Done